### ISSUE TITLE
Super slow building?

### ISSUE THREAD

**Issue by xinzheyang** [JuliaIO/HDF5.jl#544]
Hi,
It takes more than 2 hours to build HDF5 on a MacBook Pro 2017. Is this expected behavior?

**Comment by guoyiteng** [comment]
https://github.com/JuliaIO/MAT.jl/issues/83

**Comment by jonathanBieler** [comment]
It's an issue with Homebrew that fails to install GCC and build it from sources instead. The Homebrew people don't guarantee the kind of usage we do of the tool, so they won't fix it. Someone in the Julia community should try to fix the formula instead.

https://github.com/Homebrew/homebrew-core/issues/27751

The fix for now is to install hdf5 manually (brew install hdf5) or build GCC.

**Comment by tamasgal** [comment]
Hmm... Installing `hdf5` via `brew install hdf5` also triggers the `gcc` compilation. I am pretty sure this was not the case before.

```
=> Installing hdf5 dependency: gcc
Warning: Building gcc from source:
  The bottle needs the Xcode CLT to be installed.
==> Downloading https://ftp.gnu.org/gnu/gcc/gcc-9.1.0/gcc-9.1.0.tar.xz
######################################################################## 100.0%
```

**Comment by jonathanBieler** [comment]
@tamasgal I think you need to install xcode commandline tools, see:

https://github.com/Homebrew/homebrew-core/issues/37678

**Comment by musm** [comment]
fixed by #555
